stack.yaml

```diff
@@ -39,7 +39,8 @@ packages:
 - '.'
 # Dependency packages to be pulled from upstream that are not in the resolver
 # (e.g., acme-missiles-0.3)
-extra-deps: []
+extra-deps:
+  - exceptions-0.10.2@sha256:de2be08eb73e1fb4115135c62f5d5f7ed9bdb9affff318687ac6cc0438b497f0
```

Comment:
Can we use just `mtl` version of `MonadError` without such additional dependency?

Reply:
Mmm let me try.

Reply:
I tried but I'm stuck using `throwError` with the custom `TxError` . It doesn't work when you mix it with `IO` unless you use monad transformers AFAIK. Specifically in this line:

```haskell
handler :: IOException -> IO [Record]
handler _ = throwError TxError
```

Got any suggestion?